src/xcode/ENA/ENA/Source/Services/Exposure Submission/ExposureSubmissionService.swift

```diff
@@ -316,7 +331,13 @@ extension ENAExposureSubmissionService {
 
 	/// This method represents a dummy method that is sent to the submission server.
 	private func _fakeSubmissionServerRequest(completion: @escaping ExposureSubmissionHandler) {
-		self.client.submit(keys: [], tan: ENAExposureSubmissionService.fakeSubmissionTan, isFake: true) { _ in
+		// swiftlint:disable vertical_parameter_alignment_on_call
```

Comment:
Same here ad above. Can this be avoided with another indentation?

Reply:
Fixed 👍